algebird-test/src/test/scala/com/twitter/algebird/SpaceSaverTest.scala

```diff
@@ -59,5 +61,26 @@ class SpaceSaverTest extends WordSpec with Matchers {
     "be consistent in its constructors" in {
       assert(SpaceSaver(10, "ha") ++ SpaceSaver(10, "ha") ++ SpaceSaver(10, "ha") === SpaceSaver(10, "ha", 3))
     }
+
+    "Correctly serialize" in {
+      //SSOne
+      verifySerialization(SpaceSaver(40, "string that is not empty"))
```

Comment:
Will do. I wanted to do it, but before I wanted to be sure the PR could be accepted 😄